tembo-pgmq-python/tembo_pgmq_python/queue.py

```diff
@@ -47,99 +48,101 @@ def __post_init__(self) -> None:
         with self.pool.connection() as conn:
             conn.execute("create extension if not exists pgmq cascade;")
 
+    @classmethod
+    def from_env(cls, env_file=".env"):
+        """Load environment variables from a .env file and create a PGMQueue instance."""
+        load_dotenv(env_file)
+
+        # Verify that the .env file is loaded correctly
+        if not os.path.exists(env_file):
+            print(f".env file not found at path: {env_file}")
+
+        return cls(
+            host=os.getenv("PG_HOST", "localhost"),
+            port=os.getenv("PG_PORT", "5432"),
+            username=os.getenv("PG_USERNAME", "postgres"),
+            password=os.getenv("PG_PASSWORD", "postgres"),
+            database=os.getenv("PG_DATABASE", "postgres"),
+        )
+
     def create_partitioned_queue(
-        self, queue: str, partition_interval: int = 10000, retention_interval: int = 100000
+        self,
+        queue: str,
+        partition_interval: int = 10000,
+        retention_interval: int = 100000,
     ) -> None:
-        """Create a new queue
-
-        Note: Partitions are created pg_partman which must be configured in postgresql.conf
-            Set `pg_partman_bgw.interval` to set the interval for partition creation and deletion.
-            A value of 10 will create new/delete partitions every 10 seconds. This value should be tuned
-            according to the volume of messages being sent to the queue.
-
-        Args:
-            queue: The name of the queue.
-            partition_interval: The number of messages per partition. Defaults to 10,000.
-            retention_interval: The number of messages to retain. Messages exceeding this number will be dropped.
-                Defaults to 100,000.
-        """
-
```

Comment:
Is there another place that this comment could live? This is important information for users of partitioned queues. I think that documentation for the arguments are useful, but also could maybe live somewhere else if it makes more sense.

Reply:
I was thinking of adding it to the README, but I forgot. If it's very important information, I believe it should be included here as a docstring.